mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -300,7 +303,8 @@ class LinearRegressionTrainingSummary private[regression] (
     predictions: DataFrame,
```

Comment:
Do not do production when creating this summary. Then we only need to remember the feature column name.